public/gallery.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+	<script src="https://cdn.tailwindcss.com"></script>
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Pet Gallery</title>
+</head>
+<body>
+
+<div class="flex border-2 border-red-500 border-solid h-48">
+	<div class="flex items-center border-2 border-gray-500 border-solid">
+		<img class="w-16 h-16" src="Assets/images/gallery/black_heart_border.svg" alt="black heart">
+		<p>Gallery</p>
+	</div>
+	<div class="flex flex-1 justify-around items-center border-2 border-green-500 border-solid">
+		<button>filters</button>
```

Comment:
i will let this pass as a first PR :)

Reply:
yeah the border color is only to have a visual idea of the page they are not gonna be there on the next PR